confluence/models/page.py

```diff
@@ -57,5 +57,15 @@ def __init__(self, json):  # type: (Dict[str, Any]) -> None
             if 'lastUpdated' in json['history']:
                 self.last_updated = PageUpdate(json['history']['lastUpdated'])
 
+        if 'version' in json:
```

Comment:
I think this is probably fine for now, but I suspect I really want an object containing a ContentRevision or something like that. No action on this review unless you want to do it